run-checks

```diff
@@ -185,6 +185,8 @@ if [ "$STATIC" = 1 ]; then
             exit 1
         fi
         unset regexp
+        echo Checking shell scripts inside spread yamls...
+        ./spread-shellcheck spread.yaml tests
```

Comment:
The spread task one used the latest version of shellcheck from snaps. Any chance we could use something more recent(-ish) here too?

Reply:
Maybe we could keep the shellcheck spread in addition to the fast check?

Reply:
If Travis supports snaps (does it?) we could `snap install` before :)